tests/functional/spec/textarea.spec.js

```diff
@@ -43,7 +43,7 @@ describe("Textarea", () => {
 
   it("The user is unable to add more characters when the limit is reached", async () => {
```

Comment:
I think this test description will need to change with this update. It no longer reflects how the textarea is supposed to work